Sources/TuistKit/ProjectGenerator/ProjectGenerator.swift

```diff
@@ -198,21 +197,42 @@ class ProjectGenerator: ProjectGenerating {
     }
 
     // swiftlint:disable:next large_tuple
-    private func loadProjectWorkspace(path: AbsolutePath) throws -> (Project, Graph, [SideEffectDescriptor]) {
-        // Load project
-        let (project, graph, sideEffects) = try loadProject(path: path)
+    private func loadProjectWorkspace(path: AbsolutePath) throws -> (Workspace, Project, Graph, [SideEffectDescriptor]) {
```

Comment:
I couldn't find a neat way to remove the code de-dupe without impacting readability :/ 

The main differences between `loadProjectWorkspace` and `loadProject` now are:

- `Workspace` isn't created in `loadProject`
- Workspace mappers aren't applied in `loadProject`